larray/core/session.py

```diff
@@ -189,6 +248,19 @@ def save(self, fname, names=None, engine='auto', display=False, **kwargs):
         display : bool, optional
             Whether or not to display which file is being worked on. Defaults
             to False.
+            
+        Examples
+        --------
+        >>> arr1, arr2, arr3 = ndtest((3, 3)), ndtest((6, 4)), ndtest((2, 8)) # doctest: +SKIP
+        >>> s = Session(arr1=arr1, arr2=arr2, arr3=arr3)                      # doctest: +SKIP
+        
+        Save all arrays
+        
+        >>> s.save('input.h5')                    # doctest: +SKIP
```

Comment:
an output named input is a bit misleading IMO (this comment applies to all subsequent save/to* examples)